operator/gitops/argocd/pipeline-service/tekton-results/base/kustomization.yaml

```diff
@@ -25,91 +26,9 @@ configMapGenerator:
       disableNameSuffixHash: true
     behavior: replace
 
-patchesJson6902:
-  - target:
-      kind: Namespace
-      name: tekton-results
-    patch: |-
-      - op: add
-        path: "/metadata/labels"
-        value:
-          argocd.argoproj.io/managed-by: openshift-gitops
-  - target:
-      kind: Service
-      name: tekton-results-api-service
-      namespace: tekton-results
-    patch: |-
-      - op: add
-        path: "/metadata/annotations"
-        value:
-          service.beta.openshift.io/serving-cert-secret-name: tekton-results-tls
-  # the secret provided by stonesoup has data fields in the the form of db.*, patch the env passed to the api to match the fields
-  # remove default values for database env since all the values are provided by the secret.
-  - target:
-      kind: Deployment
-      name: tekton-results-api
-      namespace: tekton-results
-    patch: |-
-      - op: remove
-        path: /spec/template/spec/containers/0/env/3/value
-      - op: add
-        path: /spec/template/spec/containers/0/env/3/valueFrom
-        value:
-          secretKeyRef:
-            key: db.name
-            name: tekton-results-database
-      - op: remove
-        path: /spec/template/spec/containers/0/env/2/value
-      - op: add
-        path: /spec/template/spec/containers/0/env/2/valueFrom
-        value:
-          secretKeyRef:
-            key: db.host
-            name: tekton-results-database
-      - op: replace
-        path: /spec/template/spec/containers/0/env/1/valueFrom/secretKeyRef/key
-        value: db.password
-      - op: replace
-        path: /spec/template/spec/containers/0/env/1/valueFrom/secretKeyRef/name
-        value: tekton-results-database
-      - op: replace
-        path: /spec/template/spec/containers/0/env/0/valueFrom/secretKeyRef/key
-        value: db.user
-      - op: replace
-        path: /spec/template/spec/containers/0/env/0/valueFrom/secretKeyRef/name
-        value: tekton-results-database
-  # we are using the tekton-results namespace, so patch the arguments to watcher
-  - target:
-      kind: Deployment
-      name: tekton-results-watcher
-      namespace: tekton-results
-    patch: |-
-      - op: replace
-        path: /spec/template/spec/containers/0/args/1
-        value: tekton-results-api-service.tekton-results.svc.cluster.local:50051
-      - op: remove
```

Comment:
@avinal Why we have to drop the securityContext capability here ?

Reply:
We don't want NET_BIND permissions, it lets pod listen on any port IIRC. Let me reference the comment where @adambkaplan suggested to remove it.

Reply:
https://github.com/openshift-pipelines/pipeline-service/pull/504#discussion_r1110324875

Reply:
Ok so in your case, it seems you were adding it explicitly (+ sign in the diff) to postgres so I understand Adam's comment.
Do we want also to explicitly remove `NET_BIND_SERVICE` in our overlay while it is available by default in upstream ?
@adambkaplan ?